tests/sentry/api/endpoints/test_organization_metrics.py

```diff
@@ -54,222 +50,29 @@ def test_feature_flag(self):
             assert response.status_code == 404
 
 
-class OrganizationMetricsTest(APITestCase):
```

Comment:
Should we maybe keep these tests, but with some mock data in Snuba?

Reply:
They pretty much do the exact same thing as the integration ones, didn't really see the need to duplicating them, and the ones that did something different I kept

Reply:
OK if the same functionality is still covered, then that makes sense.